### ISSUE TITLE
Custom element doesn't exist: mini-graph-card.

### ISSUE THREAD

**Issue by domoZAM** [kalkih/mini-graph-card#553]
Hello, I'm triing to use the mini-graph  card but when I go to adding the new card in manual I get this message:

Custom element doesn't exist: mini-graph-card.
type: 'custom:mini-graph-card'

(i restarted home assistamt and cleaned also more time the cache )
this my config (on raspbian) of Home assistant:

Version | 2021.1.4
-- | --
Installation Type | Home Assistant Core
Development | false
Supervisor | false
Docker | false
Virtual Environment | true
Python Version | 3.8.6
Operating System Family | Linux
Operating System Version | 5.4.83-v7l+
CPU Architecture | armv7l

I installed mini-graph via HACS, and defined the resource for lovelace as:

url: /hacsfiles/mini-graph-card/mini-graph-card-bundle.js?v=0.10.0
type javascript module

the phisical path of the file is:
     .homeassistant/www/community/mini-graph-card/mini-graph-card-bundle.js

(i verified that browsing https://[IP-ADD:PORT]/hacsfiles/mini-graph-card/mini-graph-card-bundle.js?v=0.10.0 I keep the answer, i verified also to insert the url without expliciting the version at the end of the url with the same result)

I tried to remove and re-installing, to edit directly the raw configuration but at the end of all these attempts i always get the same answer:
Custom element doesn't exist: mini-graph-card.

Any suggestion on how to investigate more?

Thank in advance.
Marco

**Comment by happy-chap** [comment]
I'm also having the same issue (and have followed the same trouble-shooting steps) and have been unsuccessful in solving it (having also tried several the other posts that have the same problem). Any ideas?

**Comment by RomRider** [comment]
Please share a screenshot of your resources config.

**Comment by happy-chap** [comment]
![Snap 2021-02-03 at 15 24 38](https://user-images.githubusercontent.com/78432558/106768869-495b8800-6634-11eb-90b5-95710bb04da1.png)

**Comment by RomRider** [comment]
`url:` shouldn't be there. It should only be `/hacsfiles/mini-graph-card/mini-graph-card-bundle.js?v=0.10.0`

**Comment by cleggton** [comment]
I'm having a similar problem.  I've installed through HACS, Configuring my dashboard by yaml.  Thus have these lines in my config.  I've also tried without the version.

`lovelace:
  mode: yaml
  resources:
    - url: /hacsfiles/mini-graph-card/mini-graph-card-bundle.js?v=0.10.0
      type: module`

and this in my lovelace config

 `      - type: custom:mini-graph-card
        name: Power meters
        icon: mdi:server
        entities:
          - sensor.greenwave_reality_inc_powernode_1_port_power
          - sensor.greenwave_reality_inc_powernode_1_port_power_2
          - sensor.greenwave_reality_inc_powernode_1_port_power_3
          - sensor.greenwave_reality_inc_powernode_1_port_power_4  `

This in my resources

![image](https://user-images.githubusercontent.com/25527565/106777563-02be5b80-663d-11eb-804d-9ec6bbe4e428.png)

Any ideas?

**Comment by zaedas** [comment]
Im also having the same problem. I followed the "tutorial" from the information page in hacs, restarted my homeassistant and i cannot add the card manually

**Comment by domoZAM** [comment]
![image](https://user-images.githubusercontent.com/76051127/107015324-e89b8f00-679c-11eb-8d31-9aa322f5a546.png)

**Comment by domoZAM** [comment]
i have this url but i tested also others solution (without version, with different path (community/mini-gragp-card/ etc..)

**Comment by domoZAM** [comment]
OK, now works !   Thanks a lot!!!

**Comment by cleggton** [comment]
weird mine is also fixed, I have made no changes, but HA has had an update

**Comment by m1dnight** [comment]
For anyone who ends up here, and the above doesn't work. I have installed this through HACS as well (HA version  2021.7.0.dev20210607) and plugin version 0.10.0. 

After enabling advanced mode for my profile I was able to see the resources for my dashboard. The URL that was there had a query parameter (I removed it and can't reproduce) along the lines of `?hash=xxxxx`. Remove the query parameter so that your URL is `/hacsfiles/mini-graph-card/mini-graph-card-bundle.js`. That fixed it for me.
